README.md

```diff
@@ -22,13 +22,33 @@
 # AGString
 
 <p align="center">
-ℹ️ Short description of AGString
+AGString is an extension for convenient working with swift String.  
 </p>
 
-## Features
+<p align="center">
+AGRegex is Wrapper of NSRegularExpression. It gives an easy and iterative way to use RegularExpression. 
+ </p>
 
-- [x] ℹ️ Add AGString features
+## Features
```

Comment:
## Features

- AGString gives 'String type' to 'Int-Index' based referencing
    
    * ❎ Get Character object with Int index
    * ❎ Get substring object with Range<Int>
    * ❎ String utility methods(zfill, countOccurence, ltrim,rtrim ...)
    
- AGRegex 
    
    * ❎ Provide NSRange extension for String
    * ❎ Get All Match Result as AGMatch structure. 
    * ❎ Support iterative way to use match result
    * ❎ Substitute matched string to other string